Controllers/ClanController.cs

```diff
@@ -711,5 +811,76 @@ public async Task<ActionResult> AllowTag([FromQuery] string tag)
 
             return Ok();
         }
+
+        [HttpPut("~/clan/clanRankingAll")]
```

Comment:
Oh you actually made admin endpoints. Please merge them into one, rename to `refreshblabla` and move to the admin controller under the `~/admin/` path

Reply:
Fixed